Dockerfile

```diff
@@ -51,6 +51,7 @@ COPY ./ironic.conf /tmp/ironic.conf
 RUN crudini --merge /etc/ironic/ironic.conf < /tmp/ironic.conf && \
     rm /tmp/ironic.conf
 
+COPY ./jinjarender.py /bin/jinjarender
```

Comment:
let's change permissions to make this executable

Reply:
It is preserving the execute bit from the source file, like the other scripts in this Dockerfile